trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -197,6 +185,13 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker, auditDetails) =
       console.info('TFM deal %s stage has been updated to %s', dealId, updatedDealStage);
     }
 
+    console.log('===2', mappedDeal);
+    await submitDealToACBS(mappedDeal);
```

Comment:
non review comment - i feel this funciton adds so so much added cognative load when you have to touch it (it's huge!)... Next time we need to address tech debt I feel this would be a great place to start